### PR TITLE
Add README to config components

### PR DESCRIPTION
**Pull request by flands**
This can be linked to receivers and exporters in a future PR.

### CODE REVIEW

File: config/configgrpc/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Example:
+
+```yaml
+exporter:
+  otlp:
+    endpoint: otelcol2:55690
+    headers:
+      test1: "value1"
+      test 2": "value 2"
```
Comment (tigrannajaryan) [inline config/configgrpc/README.md:37]:
```suggestion
      "test 2": "value 2"
```

File: config/confighttp/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Example:
+
+```yaml
+exporter:
+  otlp:
+    endpoint: otelcol2:55690
+    headers:
+      test1: "value1"
+      test 2": "value 2"
```
Comment (tigrannajaryan) [inline config/confighttp/README.md:31]:
```suggestion
      "test 2": "value 2"
```

File: config/confighttp/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+[Exporters](https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/README.md)
+leverage client configuration.
+
+Note that client configuration supports TLS configuration, however
+configuration parameters are not defined under `tls_settings` like server
+configuration. For more information, see [configtls
+README](../configtls/README.md).
+
+- `endpoint`: Valid value syntax available [here](https://github.com/grpc/grpc/blob/master/doc/naming.md)
```
Comment (tigrannajaryan) [inline config/confighttp/README.md:17]: Wrong link. endpoint is a URL.

File: config/confighttp/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- `headers`: name/value pairs added to the request
```
Comment (tigrannajaryan) [inline config/confighttp/README.md:18]:
```suggestion
- `headers`: name/value pairs added to the HTTP request headers
```

File: config/confighttp/README.md (hunk 3 above)
Comment (flands) [inline config/confighttp/README.md:17]:
```suggestion
- `endpoint`: address:port
```

**Comment by flands** [comment]
@tigrannajaryan all three doc PRs should be ready -- thanks for the reviews.

**Review by bogdandrutu — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
@flands please fix the conflicts.

**Comment by tigrannajaryan** [comment]
@flands build failed, looks like there are broken links.

**Pull request merged**